conversions/SnowflakeTimestamp_Fn.py

```diff
@@ -0,0 +1,8 @@
+# Discord save the times at which a message is sent as "Snowflake", this is a code to convert the Snowflake into the actual timestamp.
+import math
+import time
+
+con = input("SnowflakeID ")
```

Comment:
This line will crash our continuous integration testing as discussed in [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).  Please do not open new pull requests in this repo until November 1st. 2022.